python/plugins/processing/tests/data/raster.tif.aux.xml

```diff
@@ -0,0 +1,20 @@
+<PAMDataset>
```

Comment:
Did you find out if we can move all the files to testdata so we can get rid of this folder?